trunk/SUAVE/Components/Energy/Converters/Gearbox.py

```diff
@@ -49,16 +49,13 @@ def __defaults__(self):
         """            
```

Comment:
Needs a header change to indicate modified date.

Reply:
Header added and date updated